src/frontend/src/utils/userNumber.test.ts

```diff
@@ -90,11 +91,11 @@ function testLocalStorage(
 ) {
   test(name, () => {
     localStorage.clear();
-    if (opts?.before !== undefined) {
+    if (nonNullish(opts) && nonNullish(opts?.before)) {
```

Comment:
why is the first check necessary?

Reply:
<img width="1470" alt="Capture d’écran 2023-04-29 à 11 30 29" src="https://user-images.githubusercontent.com/16886711/235295659-9d258e06-abc6-4e3b-a8ef-69fe52821f19.png">